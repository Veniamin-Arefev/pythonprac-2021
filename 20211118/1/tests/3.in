@objcount
class C:
    def __init__(self):
        self.kek = 5

    def __del__(self):
        print("Lol")

c, d, e = C(), C(), C()
print(C.counter)
c = 100500
del e
print(C.counter)