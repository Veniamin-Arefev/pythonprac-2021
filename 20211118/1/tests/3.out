3
Lol
Lol
1
Lol
